test/media/streaming_engine_unit.js

```diff
@@ -2743,36 +2802,41 @@ describe('StreamingEngine', () => {
       messageData: new Uint8Array([0x74, 0x65, 0x73, 0x74]),
     };
 
+    // Different box version, same contents.
+    const emsgObjV1 = emsgObjV0;
+
     beforeEach(() => {
-      setupVod();
+      // setup an offset for the timestamps.
+      setupVod(false, 10);
       mediaSourceEngine = new shaka.test.FakeMediaSourceEngine(segmentData);
       createStreamingEngine();
     });
 
+    function setSegment0(emsgBox) {
+      segmentData[ContentType.VIDEO].segments[0] = emsgBox;
+    }
+
     it('raises an event for registered embedded emsg boxes', async () => {
-      segmentData[ContentType.VIDEO].segments[0] = emsgSegment;
-      videoStream.emsgSchemeIdUris = [emsgObj.schemeIdUri];
+      setSegment0(emsgSegmentV0);
+      videoStream.emsgSchemeIdUris = [emsgObjV0.schemeIdUri];
 
       // Here we go!
       streamingEngine.switchVariant(variant);
       streamingEngine.switchTextStream(textStream);
+      console.log('Skipping native screenshot tests on Safari < 16');
```

Comment:
this log has nothing to do with this test.

Reply:
I must have accidentally pasted it here and overlooked it.  Thanks!